src/features/content-insights/CompactCount.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { injectIntl, IntlShape } from 'react-intl';
+import classNames from 'classnames';
+
+import numAbbr from '../../utils/numAbbr';
+
+interface Props {
+    className?: string;
```

Comment:
minor: not sure if it's needed, but should we provide a default value (i.e. '') for className in the case that one isn't provided?

Reply:
`classNames` gracefully handles falsey values when generating the class name